Server/src/lib/envHelper.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+const appSettings = require('../../appsettings.json');
+
+export const getResourceConnectionString = (): string => {
+  const resourceConnectionString = process.env['ResourceConnectionString'] || appSettings.ResourceConnectionString;
+
+  if (!resourceConnectionString) {
+    throw new Error('No ACS connection string provided');
+  }
+
+  return resourceConnectionString;
+};
+
+export const getEnvUrl = (): string => {
```

Comment:
suggesstion: rename to `getEndpointUrl()`

Reply:
Created an ADO item
https://skype.visualstudio.com/SPOOL/_workitems/edit/2691361